x-pack/packages/kbn-langchain/server/language_models/constants.ts

```diff
@@ -5,11 +5,17 @@
  * 2.0.
  */
 
-export const getDefaultArguments = (llmType?: string, temperature?: number, stop?: string[]) =>
+export const getDefaultArguments = (
+  llmType?: string,
+  temperature?: number,
+  stop?: string[],
+  maxTokens?: number
+) =>
   llmType === 'bedrock'
     ? {
         temperature: temperature ?? DEFAULT_BEDROCK_TEMPERATURE,
         stopSequences: stop ?? DEFAULT_BEDROCK_STOP_SEQUENCES,
+        maxTokens,
```

Comment:
`maxTokens` is already defaulted in the connector so we don't need to default it here